edge-modules/functions/README.md

```diff
@@ -1,38 +1,43 @@
 # Function Sample
```

Comment:
Looking at this file more carefully, I think I agree with your first impression...we might just want to remove it. The doc you referenced (https://docs.microsoft.com/en-us/azure/iot-edge/tutorial-deploy-function) already describes what you need to do, in a more generalized and user-friendly way.  :)